### DISCUSSION TITLE
How to create dropdown parameter filter

### DISCUSSION THREAD

**Discussion by pcosta007** [apache/superset#19820] [Q&A / Help]
Hi,

Is there a way to create a parameter with multiple user defined values that could be used as a filter selection? The parameter would be used within if statement calculations to create a dynamic calculated field. For example, if you had a field called sales, create a parameter whose values are 'min','max','median', and a calculated field where if parameter = min then min(sales) elif param = max then max(sales), etc. The dashboard user could then select the parameter value from a dropdown to adjust all dashboard elements that use that calculation. 

Tableau and AWS Quicksight offer this functionality, but I've been unable to find something similar with Superset. Is there a way to do this or if not is it in a future roadmap?
<img width="222" alt="Screen Shot 2022-04-22 at 9 24 38 AM" src="https://user-images.githubusercontent.com/104209303/164723122-3c2f8d2e-00f5-4b54-ad81-683695fe8873.png">

<img width="538" alt="Screen Shot 2022-04-22 at 9 25 35 AM" src="https://user-images.githubusercontent.com/104209303/164723277-55b6157f-c3a6-474d-9bb0-44a2e3f0193c.png">

**Comment by mursidehsani** [comment]
I need something similar too. To create a parameter to choose date, like a drop down of choosing day, month, year etc.

**Comment by iamzoltan** [comment]
I would also like something similar, any traction on this?
